blueprints/instance-initializer-test/files/tests/unit/instance-initializers/__name__-test.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import { initialize } from '<%= dependencyDepth %>/instance-initializers/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+
+var appInstance, application;
+
+module('<%= friendlyTestName %>', {
+  beforeEach: function() {
+    Ember.run(function() {
+      application = Ember.Application.create();
+      appInstance = application.__deprecatedInstance__;
```

Comment:
Nothing in particular, I was copying initializer tests and just changed this line 
`appInstance = application.__deprecatedInstance__;`

I already started a PR in ember-test-helpers to create a `moduleForInstanceInitializer` that hides this details, but it's not a trivial thing (all that black magic with the register is pretty obscure). I have to find some time to finish it.